schema.graphql

```diff
@@ -1,25 +1,43 @@
 type Payment @entity {
+  "hash transaction of the payment"
   id: ID!
+  "network contract address"
   contractAddress: Bytes!
+  "contract of token used for payment"
   tokenAddress: Bytes
+  "address payment was made from"
   to: Bytes!
+  "address payment was made to"
   from: Bytes!
+  "is the reference data used to track the transfer"
   reference: Bytes!
+  "block payment took place"
   block: Int!
+  "time stamp of payment"
   timestamp: Int!
+  "transaction hash of payment"
   txHash: Bytes!
+  "gas used for payment"
   gasUsed: BigInt!
+  "gas fee at time of payment"
   gasPrice: BigInt!
+  "amount of payment"
   amount: BigDecimal!
 
   # fee values
+  "fee charged for payment"
   feeAmount: BigDecimal
+  "address of person paying fee"
```

Comment:
I suspect this is incorrect and should be the "address where the fee is sent", but I need to confirm